index.html

```diff
@@ -58,12 +61,347 @@ <h1>
     
             </div>
 
-        </section>
+        </section>     
+    </section>
+    <section class="recent-works">
+      <h1>My Recent Works</h1>
+      <div class="line-rule"></div>
+
+      <div class="img-placeholder">
+        <img src="assets/img-placeholder.png" alt=""/>
+      </div>
+
+      <div class="post">
+        <h2>Multi-Post Stories</h2>
+      </div>
+      <div class="content">
+        <h4>
+          A daily selection of privately personalized reads; 
+          no accounts or sign-ups required. 
+          has been the industry's standard dummy text ever since the 1500s, 
+          when an unknown printer took a standard dummy text.
+        </h4>
+      </div>
+      <div class="tag">
+        <h6>Css</h6>
+        <h6>Html</h6>
+        <h6>bootstrap</h6>
+        <h6>JavaScript</h6>
+      </div>
+      <div class="project-btn">
+        <h4>See Project</h4>
+      </div>      
+      <section class="projects">
 
         
+        <h1>Profesional Art Printing Data</h1>
+        <small>A daily selection of privately personalized reads; 
+          no accounts or sign-ups required. 
+          has been the industry's standard
+        </small>
+        <div class="languages">
+          <div class="project-lang">
+            <h5>html</h5>
+          </div>
+          <div class="project-lang2">
+            <h5>bootstrap</h5>
+          </div>
+          <div class="project-lang3">
+            <h5>Ruby</h5>
+          </div>
+        </div>
+
+        <div class="action">
+          <h3>See Project</h3>
+        </div>
+
+
+
+
         
 
+      </section>
+      <section class="projects">
+
         
+        <h1>Profesional Art Printing Data</h1>
+        <small>A daily selection of privately personalized reads; 
+          no accounts or sign-ups required. 
+          has been the industry's standard
+        </small>
+        <div class="languages">
+          <div class="project-lang">
+            <h5>html</h5>
+          </div>
+          <div class="project-lang2">
+            <h5>bootstrap</h5>
+          </div>
+          <div class="project-lang3">
+            <h5>Ruby</h5>
+          </div>
+        </div>
+
+        <div class="action">
+          <h3>See Project</h3>
+        </div>
+
+
+
+
+        
+
+      </section>
+      <section class="projects">
+
+        
+        <h1>Profesional Art Printing Data</h1>
+        <small>A daily selection of privately personalized reads; 
+          no accounts or sign-ups required. 
+          has been the industry's standard
+        </small>
+        <div class="languages">
+          <div class="project-lang">
+            <h5>html</h5>
+          </div>
+          <div class="project-lang2">
+            <h5>bootstrap</h5>
+          </div>
+          <div class="project-lang3">
+            <h5>Ruby</h5>
+          </div>
+        </div>
```

Comment:
- [ ] Kindly position your language elements correctly to match the original design.

![image](https://user-images.githubusercontent.com/98453979/205311461-d30fbedf-8e06-47e7-b469-773c10a25201.png)

### Checkout Figma

![image](https://user-images.githubusercontent.com/98453979/205311598-d3639d18-6407-4dfa-a365-dcc0f082f44c.png)